quality_control/static_checks/check_black.py

```diff
@@ -5,10 +5,10 @@
 # pylint: disable=duplicate-code
 from pathlib import Path
 
-from config.cli_unifier import _run_console_tool, choose_python_exe, handles_console_error
-from config.console_logging import get_child_logger
-from config.constants import PROJECT_CONFIG_PATH, PROJECT_ROOT
-from config.project_config import ProjectConfig
+from quality_control.cli_unifier import _run_console_tool, choose_python_exe, handles_console_error
```

Comment:
1. move one check_black one level up
2. make all imports like `from constants import ...` - quality control should not be present